op-challenger/config/config.go

```diff
@@ -32,10 +32,30 @@ type TraceType string
 const (
 	TraceTypeAlphabet TraceType = "alphabet"
 	TraceTypeCannon   TraceType = "cannon"
+
+	// Devnet game IDs
+	DevnetGameIDAlphabet = uint8(0)
```

Comment:
Feels kind of weird that different networks have different ids

Reply:
See comment above from @trianglesphere, ticket created @ https://linear.app/optimism/issue/CLI-4369/consider-having-game-types-be-the-same-across-all-networks